superbit_lensing/shear_profiles/make_annular_catalog.py

```diff
@@ -332,8 +332,8 @@ def run(self, overwrite=False, vb=False):
             p = f'{self.run_name}_'
         else:
             p = ''
-        outfile = os.path.join(self.outdir, f'{p}_annular_shear_tab.fits')
-        plotfile = os.path.join(self.outdir, f'{p}_tan_shear.pdf')
+        outfile = os.path.join(self.outdir, f'{p}shear_profile.fits')
```

Comment:
Would you be OK with making outfile something like `shear_profile_cat.fits` and the plot `shear_profile_plot.pdf` or at least `outfile = shear_profile.fits`, `plotfile = shear_profile_plot.pdf`? That way, it's obvious that the pdf is a plot of the information contained in the FITS file.

Reply:
I suppose I wasn't that worried about it as the extension tells you what it is. But I agree that the names aren't great. How about `{run_name}_shear_profile_cat.fits` and `shear_profile.pdf`?